src/nvim/fileio.c

```diff
@@ -6273,23 +6296,27 @@ bool has_event(int event) FUNC_ATTR_PURE FUNC_ATTR_WARN_UNUSED_RESULT
   return first_autopat[event] != NULL;
 }
 
-static int 
-apply_autocmds_group (
-    event_T event,
-    char_u *fname,         /* NULL or empty means use actual file name */
-    char_u *fname_io,      /* fname to use for <afile> on cmdline, NULL means
-                               use fname */
-    int force,                  /* when TRUE, ignore autocmd_busy */
-    int group,                  /* group ID, or AUGROUP_ALL */
-    buf_T *buf,           /* buffer for <abuf> */
-    exarg_T *eap           /* command arguments */
-)
+/// Execute autocommands for "event" and file name "fname".
+///
+/// @param event event that occured
+/// @param fname filename, NULL or empty means use actual file name
+/// @param fname_io filename to use for <afile> on cmdline,
+///                 NULL means use `fname`.
+/// @param force When true, ignore autocmd_busy
+/// @param group autocmd group ID or AUGROUP_ALL
+/// @param buf Buffer for <abuf>
+/// @param exarg Ex command arguments
+///
+/// @return true if some commands were executed.
+static bool apply_autocmds_group(event_T event, char_u *fname, char_u *fname_io,
+                                 bool force, int group, buf_T *buf,
+                                 exarg_T *eap)
```

Comment:
This name doesn't match the `@param` above. I don't want to rebase this PR, but maybe you could sneak this change into another PR?